platform/client-admin/src/forms/create-volunteer.ts

```diff
@@ -0,0 +1,61 @@
+import SimpleSchema2Bridge from 'uniforms-bridge-simple-schema-2';
+import SimpleSchema from 'simpl-schema';
+
+const volunteerForm = new SimpleSchema({
+    firstName: {
+        type: String,
+        max: 120
+    },
+    lastName: {
+        type: String,
+        max: 120
+    },
+    email: {
```

Comment:
is it possible with uniforms to say that this input is an email field e.g `<input type="email">`? @kingsleyzissou

Reply:
It is in the uniforms but not sure if that is mapped in the lib.

Reply:
adding `uniforms: {type:'email'}` fixed it - thanks @kingsleyzissou